mix.exs

```diff
@@ -109,12 +111,13 @@ defmodule DotCom.Mixfile do
       {:logster, "1.1.1"},
       {:mail, "0.3.1"},
       {:mock, "0.3.8", [only: :test]},
+      {:mox, "1.1.0", [only: :test]},
```

Comment:
These seem to be added as dependencies but not used in this pull request

Reply:
The commit message that added these was `[fixup merge commit](https://github.com/mbta/dotcom/pull/1897/commits/3313b4834a0fe3dc655a30dacc793d3793a98501)`....these were supposed to be in the `feat/trip-planner-improvements` branch (and are being used there already).